hieradata/common.yaml

```diff
@@ -1485,6 +1485,7 @@ router::assets_origin::asset_routes:
   '/government/assets/': "whitehall-frontend"
   '/government/placeholder': "whitehall-frontend"
   '/government/uploads/': "whitehall-frontend"
+  '/government/uploads/system/uploads/organisation/logo': "static"
```

Comment:
> Some of the path patterns have a trailing forward-slash, but others do not. Do you know why? And are you confident that having no trailing forward-slash is correct?

I've added a trailing slash to this new path but not looked at the other paths to know whether they should or shouldn't have trailing slashes. The lack of trailing slash meant that, for example, `.../logo-foo` would've been matched by this new location. Adding the trailing slash prevents this and ensures that the trailing slash has to be present in the request URI. In practice, I'm not sure it'll make any real difference but I think adding the slash makes the intention clearer. Good spot!

Reply:
> Given you've tested this on your development VM I assume the router matches this new route in preference to '/government/uploads/'? Do you know how come this work? Is it because the router matches longer routes in preference to shorter ones? I think it might be worth explaining this in the commit note.

I'm fairly certain that requests for these assets don't go through the router - they just go through nginx. As such, it's the nginx behaviour that prefers a longer match to a shorter match when calculating the location block to use. I've updated the commit message to explain this:

> Note that Nginx matches longer prefix matches in preference to shorter
> matches[1], which is why requests for organisations logos will be
> handled by our new location block instead of the `/government/uploads/`
> block that was already in place.